src/Tizen.NUI/src/public/BaseComponents/VectorGraphcis/Drawable.cs

```diff
@@ -0,0 +1,116 @@
+/*
+* Copyright(c) 2021 Samsung Electronics Co., Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI.BaseComponents.VectorGraphics
+{
+    /// <summary>
+    /// Drawable is a object class for drawing a vector primitive.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Drawable : BaseHandle
+    {
+        /// <summary>
+        /// Creates an initialized drawable.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private Drawable() {}
+
+        internal Drawable(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// The transparency level [0 ~ 1.0], 0 means totally transparent, while 1 means opaque.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float Opacity
+        {   
+            get {
```

Comment:
could you change this as like this?
`get`
`{
...`
you can use auto code editing by "ctrl+k+d" or "ctrl+k+f" in Visual Studio.

could you check the sites below for easy way of editing code?

- Visual Studio in Windows : https://stackoverflow.com/questions/1435691/visual-studio-format-entire-file

- VS Code in Ubuntu : https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
Thank you for tip.